app/client/components/BaseView.js

```diff
@@ -581,7 +581,8 @@ BaseView.prototype._saveEditRowField = function(editRowModel, colName, value) {
       // Display this rowId, even if it doesn't match the filter
       .then((result) => {
         if (!this.isDisposed()) {
-          this._sectionFilter.addTemporaryRow(rowId);
+          const colType = this.currentColumn().pureType();
+          this._sectionFilter.addTemporaryRow(rowId, colType);
```

Comment:
Other code also calls `SectionFilter.addTemporaryRow`, so I think we'll want to do the check outside of it. I'd suggest doing this instead:

```
if (!this.isDisposed() && this.currentColumn().pureType() !== 'Bool') {
  this._sectionFilter.addTemporaryRow(rowId);
}
```

Reply:
Sounds good :+1: